src/config/ethereum.ts

```diff
@@ -0,0 +1,7 @@
+import { providers } from "ethers"
+
+// TODO: Set correct Infura url.
+const INFURA_URL =
```

Comment:
I started a thread to chase this ID down and get some opinions about how to inject this.